app/src/main/AndroidManifest.xml

```diff
@@ -1,91 +1,82 @@
+
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/Theme.AppCompat.Light.NoActionBar" >
+        android:theme="@style/MyMaterialTheme">
         <activity
             android:name=".SplashScreenActivity"
-            android:label="@string/app_name" >
+            android:label="@string/app_name">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".MainActivity"
-            android:label="@string/app_name"></activity>
-        <activity
-            android:name="com.peacecorps.pcsa.reporting.HomeScreen"
-            android:label="@string/app_name"
-            android:theme="@style/Theme.AppCompat.Light.NoActionBar" >
-        </activity>
         <activity
-            android:name="com.peacecorps.pcsa.reporting.Types"
-            android:label="@string/title_activity_types" >
-        </activity>
-        <activity
-            android:name="com.peacecorps.pcsa.reporting.Steps"
-            android:label="@string/title_activity_steps" >
-        </activity>
+            android:name=".MainActivity"
+            android:label="@string/app_name" />
         <activity
-            android:name="com.peacecorps.pcsa.reporting.FAQ"
-            android:label="@string/title_activity_faq" >
-        </activity>
+            android:name=".reporting.HomeScreen"
+            android:label="@string/app_name"
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar"></activity>
         <activity
-            android:name="com.peacecorps.pcsa.reporting.Confidentiality"
-            android:label="@string/title_activity_confidentiality" >
-        </activity>
+            android:name=".reporting.Types"
+            android:label="@string/title_activity_types"></activity>
         <activity
-            android:name="com.peacecorps.pcsa.reporting.Resources"
-            android:label="@string/title_activity_resources" >
-        </activity>
+            android:name=".reporting.Steps"
+            android:label="@string/title_activity_steps"></activity>
         <activity
-            android:name=".get_help_now.ContactPostStaff"
-            android:label="@string/title_activity_contact_post_staff" >
-        </activity>
+            android:name=".reporting.FAQ"
+            android:label="@string/title_activity_faq"></activity>
         <activity
-            android:name=".get_help_now.ContactOtherStaff"
-            android:label="@string/title_activity_contact_other_staff" >
-        </activity>
+            android:name=".reporting.Confidentiality"
+            android:label="@string/title_activity_confidentiality"></activity>
         <activity
-            android:name=".get_help_now.OtherStaffContent"
```

Comment:
Not Activity anymore!
Have been converted to fragments, hence this change. You can also see similar changes for ContactPostStaff, OtherStaffContent